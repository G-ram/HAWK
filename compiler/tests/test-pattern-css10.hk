BEGIN{}
[@a[href^="http://www.wikicu.com/Main_Page"]@]{
    print(this);
}
END{}
