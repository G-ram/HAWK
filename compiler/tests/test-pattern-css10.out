rel=nofollow
href=http://www.wikicu.com/Main_Page
innerhtml=Main page
nextChild=
id=